test/screenshot.spec.ts

```diff
@@ -50,26 +50,8 @@ describe('Screenshots', function () {
       });
       expect(screenshot).toBeGolden('screenshot-clip-rect.png');
     });
-    // TODO: enable after the screenshot is fixed.
```

Comment:
Why is this test being removed?

Reply:
The bug https://crbug.com/1173457 was closed with the motivation: if you need a screenshot bigger than the ViewPort, you have to use the flag `captureBeyondViewport `.

This change uses the flag, screenshot works in the new way, but correctly. No further `TODO` needed.

It used to be 2 similar tests with the different expectations:
* skipped `should clip elements to the viewport size without artefacts`
* `should clip elements to the viewport size with artefacts`

I removed one of them and adjusted the expectations. It covers the issue point and the breaking change, verifying the screenshot works correctly.

Reply:
Ack, thanks!